TWLight/emails/tasks.py

```diff
@@ -324,7 +324,7 @@ def send_approval_notification_email(instance):
 
     # If, for some reason, we're trying to send an email to a user
     # who deleted their account, stop doing that.
-    if instance.editor:
+    if instance.editor and instance.partner.authorization_method!=1:
```

Comment:
This will throw an error when we approve applications for partners with an authorization method other than `CODES`. That seems like a problem to me. Also, when you refer to the `authorization_method` please use the choice label instead of the number for legibility. You can just search the codebase for `Partner.CODES` to see plenty of examples.

Reply:
Yes, that was mistaken feedback based on a readthrough. It looks as if the error would be logged if the auth type *is* `codes`. You've achieved the goal of not sending a email here for `codes`, but you are doing it in a guard clause that is designed to log errors and send the dev team email alerts. If you check the error output in docker, you'll see that where the current codebase sends the extra email to a user, your code is now logging an error incorrectly.